src/Microsoft.Azure.SignalR.Common/Auth/JwtSecurityTokenHandlerSignalR.cs

```diff
@@ -0,0 +1,122 @@
+﻿/*------------------------------------------------------------------------------
+ * Simplified from https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/6.22.0/src/System.IdentityModel.Tokens.Jwt/JwtSecurityTokenHandler.cs#L487
+ * Compared with original code:
+ *      1. Remove useless code
+ *      2. New class `JwtSecurityTokenHandlerSignalR` inherited from `JwtSecurityTokenHandler`
+ *        Reason: Many removeds code in `JwtSecurityTokenHandler` are used in unit tests, if we still overwrite the class with the name`JwtSecurityTokenHandler` rather than creating a new class inhertied from `JwtSecurityTokenHandler`, we had to preserve useless codes that have nothing to do with JWT token generation. To make these useless codes work, we had to introduce more and more source codes from the original package.
```

Comment:
I think.... UnitTest should use `JwtSecurityTokenHandler` from  `System.IdentityModel.Tokens.Jwt` instead, so that we can at least make sure our code does not have regressions.

Reply:
fixed